docs_src/getting-started/index.md

```diff
@@ -8,21 +8,11 @@ Developer/Contributor.
 If you want to get the EdgeX platform and run it (but do not
 intend to change or add to the existing code base now) then you
 are considered a "User". You will want to follow the
-[Getting Started Users](./Ch-GettingStartedUsers.md) guide. The
-Getting Started Users guide takes you through the process of getting
-the latest release EdgeX Docker Containers from Docker Hub. If you wish
-to get the latest EdgeX containers (those built from the current ongoing
-development efforts prior to release), then see
-[Getting Started Users - Nexus](./Ch-GettingStartedUsersNexus.md). 
```

Comment:
The sentence about latest images in nexus is moved to `getting-started/Ch-GettingStartedDevelopers.md`. Users who do need latest builds of the images will go to Getting Started as a User -> Docker and find out about it.